File: 프로그래머스/0/181909. 접미사 배열/접미사 배열.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public string[] solution(string my_string) 
    {
        List<string> str = new List<string>();
        string[] answer = new string[my_string.Length];
        
        for (int i=0; i<=my_string.Length-1; i++) {
            str.Add(my_string.Substring(i, my_string.Length-i));   
        }
        str.Sort();
        answer = str.ToArray();
        return answer;
    }
}

File: 프로그래머스/0/181909. 접미사 배열/접미사 배열.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<string> solution(string my_string) 
{
    vector<string> answer;
    
    int start = my_string.length() - 1;
    vector<string> strVec;
    int len = 1;
    for(int i=start; i>=0; i--)
    {
        strVec.push_back(my_string.substr(i,len));
        len++;
    }
    sort(strVec.begin(), strVec.end());
    return strVec;
}